### PR TITLE
Audio mp3

### PR DESCRIPTION
**Pull request by norkans7**
depends on https://github.com/rapidpro/dash/pull/49

### CODE REVIEW

**Comment by nicpottier** [comment]
Screenshot of what it looks like?

**Comment by norkans7** [comment]
mp3player

File: templates/public/story_read.haml [hunk 1]
```diff
@@ -128,8 +136,33 @@
...
+    .audiojs {
+      width: 100%;
+    }
+
+    .audiojs .play-pause {
+      min-width: 40px;
+    }
+
+    .audiojs .scrubber {
+      min-width: calc(100% - 170px);
```
Comment (norkans7) [inline templates/public/story_read.haml]: this css makes it responsive

**Comment by nicpottier** [comment]
Nice, looks good.

**Pull request merged**